sync/azuredatabricks.py

```diff
@@ -292,11 +292,6 @@ def _get_cluster_instances(cluster: dict) -> Response[dict]:
         compute = _get_azure_client(ComputeManagementClient)
         if resource_group_name:
             vms = compute.virtual_machines.list(resource_group_name=resource_group_name)
-
-            for vm in vms:
-                compute.virtual_machines.instance_view(
-                    resource_group_name=resource_group_name,
-                )
```

Comment:
Don't know why this was here in the first place, but the vms object can only be iterated through once so it even after fixing the `instance_view` call it would cause cluster instances to be empty